OneCommunityGlobal/HGNRest
# Shengwei fix jae account update permission issue

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/69882989/f94b3008-1f03-4358-8c64-3a38507f1d2b)

## Related PRS (if any):
This PR is related to the FE [PR1681](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1681)

…

## Main changes explained:
- Update` src/controllers/badgeController.js` to modify how a new badge collection is created and to match the date format for the last modified field in the badge collection schema.
- Create a generic exception handler file that logs exceptions to Sentry and logs error stack traces while in local development.

## How to test:
1. check into the current branch and FE [PR1681](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1681)
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Refer to FE [PR1681](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1681)


## Note:
I will create a follow-up PR to address the edit restriction on Jae's account in the badge management component. The bug related to incorrect badge assignment is causing a block.
